lib/worker.test.js

```diff
@@ -83,6 +86,40 @@ describe('Worker', () => {
     await conn.close();
   });
 
+  afterAll(async () => {
+    await worker.finish();
+  });
+
+  test('.type throws error if it is not defined', async () => {
+    await expect(() => worker.type).toThrow();
+  });
+
+  test('start() throws error if type is not defined', async () => {
+    expect.assertions(1);
+
+    worker.type = '';
+
+    try {
+      await worker.start();
+    } catch (e) {
+      expect(e).toEqual(
```

Comment:
```suggestion
          await expect(worker.start()).rejects.toThrow(
      new Error(
        'Worker type is not defined! Implement `get type()` fucntion.'
      )
    );
```
давай так попробуем? чуть читабельнее выглядит